apiserver/facades/client/modelconfig/register.go

```diff
@@ -40,9 +41,12 @@ func makeFacade(ctx facade.ModelContext) (*ModelConfigAPI, error) {
 
 	configService := domainServices.Config()
 	modelSericve := domainServices.ModelInfo()
+	controllerTag := names.NewControllerTag(ctx.ControllerUUID())
+
 	return NewModelConfigAPI(
 		ctx.ModelUUID(),
 		ctx.State(),
+		controllerTag,
```

Comment:
The pattern we've been using is to record the controller uuid and when needed, do `names.NewControllerTag(c.controllerUUID)`